packages/@react-aria/calendar/src/useCalendarBase.ts

```diff
@@ -184,8 +143,6 @@ export function useCalendarBase(props: CalendarPropsBase & DOMProps, state: Cale
       'aria-disabled': isDisabled || null,
       'aria-labelledby': labelProps['aria-labelledby'],
       'aria-describedby': selectedDateDescription ? captionId : null,
-      'aria-colcount': 7,
-      'aria-rowcount': state.weeksInMonth + 1,
```

Comment:
I don't think any of these aria count/index properties here and on the rows/cells are needed right? The browser should compute them based on the DOM hierarchy automatically. We're not doing any virtualization of the calendar so should be ok?

Reply:
hmm yeah possibly? though I got the impression from reading https://www.w3.org/TR/wai-aria-practices-1.2/#gridAndTableProperties_rows that it wasn't necessary even with divs unless all rows are not in the dom.